src/wolfsftp.c

```diff
@@ -1577,6 +1553,19 @@ int wolfSSH_SFTP_CreateStatus(WOLFSSH* ssh, word32 status, word32 reqId,
 }
 
 
+static int GetAndCleanPath(const char* dP, const byte* data, word32 sz,
```

Comment:
Add comments explaining what this function does. dP is also not a descriptive name.

Reply:
Renamed dP and commented.